src/scripts/main.js

```diff
@@ -22,6 +22,19 @@ const tree = document.querySelector('#tree');
 
 function createTree(element, data) {
   // WRITE YOUR CODE HERE
+  const ul = document.createElement('ul');
+
+  for (const key in data) {
+    const li = document.createElement('li');
+
+    li.textContent = key;
+
+    if (Object.keys(data[key]).length > 0) {
+      createTree(li, data[key]);
+    }
+    ul.appendChild(li);
+  }
+  element.appendChild(ul);
 }
```

Comment:
The function 'createTree' is a recursive function that creates a nested list (ul/li structure) based on the nested object passed as the 'data' parameter. However, it seems that it does not actually add the nested lists to the parent 'li' elements. In line 33, you're calling 'createTree' with the current 'li' element and the nested object, but you're not doing anything with the returned value. You should append the returned 'ul' element to the current 'li' element.